roles/ocp_on_libvirt/templates/hosts.j2

```diff
@@ -28,6 +28,10 @@ enable_redfish=true
 externalMACAddress={{ externalMACAddress }}
 {% endif %}
 
+{% if ssh_extra_keys_paths is defined and ssh_extra_keys_paths %}
```

Comment:
LGTM, just curious in the second part of the if condition `and ssh_extra_keys_paths` what exactly validates? that the length of the var is not empty?